guidelines/ambassador-allontanamento.md

```diff
@@ -0,0 +1,34 @@
+# Allontanamento Ambassador
+
+Questo documento è stato redatto per fornire una guida al team Ambassador per l'allontanamento di un membro del team.
+
+## Motivi di allontanamento
+
+In tutti i casi sotto elencati, per far partire il sondaggio è sufficiente che un membro del team Ambassador ne faccia richiesta al Communications Group indicando l'evidenza e la regola violata.
+
+Quando viene usato il termine _sondaggio standard infra-gruppo_ si intende un sondaggio con quorum al 50% e maggioranza semplice della durata di 48 ore tra i membri del gruppo di riferimento.
+
+Il mancato raggiungimento del quorum in un sondaggio standard infra-gruppo corrisponderà all'apertura del medesimo sondaggio, con le medesime condizioni, rivolto a tutti i membri del team Ambassador.
+
+Un membro del team Ambassador può essere allontanato per i seguenti motivi:
+
+- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
+- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.
+- Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.
```

Comment:
Su questo punto non concordo, per me un Ambassador può non prendere in assegnazione un topic specifico, ma può essere attivo nella scrittura di alcuni senza esserne OWNER! 

Nella mia visione l'ambassador è un contributor molto attivo nel progetto e con attivo intendo che scrive cose, commenta issue e fa review!

Questo rende molto aperto il fatto di essere Ambassador che è ottimo per una community Open, poi solo se vuole ulteriori incarichi può diventare OWNER di un capitolo (andrebbe definito come diventare OWNER) o entrando in un wg!

Reply:
Io vedo l'owner di un capitolo come persona responsabile dello stesso, che sia o meno da sola nel farlo, per quello indicavo la necessità..
Possiamo anche metterci in 7 sul testing e contribuire tutti assieme, ma ognuno dei 7 sarà assegnatario dell'argomento.

Reply:
Quindi tu vedi due tipi di Ambassador su un argomento, uno che faccia da owner e uno che faccia da contributor? Non è troppo complicata?

Reply:
In realtà è più semplice... Ambassador è uno che contribuisce in modo continuativo, poi se e solo se la sente si prende responsabilità aggiuntive diventando OWNER di un capitolo o entrando in un WG.

Io mi immagino una cosa molto semplice, una persona che si diverte a contribuire in maniera utile al progetto, ma non se la sente di avere ruoli più complessi per tempo o più semplicemente perché non se la sente, però mi pare giusto che gli venga riconosciuto un lavoro molto spinto sul progetto.

Lo vedo come modo di rendere più inclusivo il tutto...

Esempio: contribuisci spesso a Kubernetes entri nella Org e puoi fare il reviewer e tutto, ma non sei obbligato a prenderti leadership di un progetto (per noi capitolo)

Reply:
Però così rischiamo il problema opposto.. Da 40 Ambassador diventiamo 500 :)

Reply:
Concordo, capisco il voler essere inclusivi e mi piace come approccio, ma non possiamo neppure gestire un numero di Ambassador troppo elevato.

@davideimola mi sembra che tu abbia espresso il concetto di "contributor molto attivo" - se non sei owner di un capitolo o non partecipi in un WG mi sembra difficile attribuire il ruolo di Ambassador, poi però ci eravamo anche proposti di inserire un ruolo di "amico del progetto", ricordi? Potrebbe essere ciò a cui ti riferisci?